calico-enterprise/reference/resources/policyrecommendations.mdx

```diff
@@ -29,6 +29,33 @@ $ kubectl patch policyrecommendationscope default -p '{"spec":{"<parameter>":"<v
 
 `$ kubectl patch policyrecommendationscope default -p '{"spec":{"interval":"5m"}}'`
 
+## Sample YAML
```

Comment:
Why is this being added?

Reply:
I need to investigate this. This is the file where I had a merge conflict.

Reply:
Fixed this.